### PR TITLE
how to get the current balance

### PR DESCRIPTION
**Pull request by felipaoo**
- Add an example of how to get the current balance in your account
- Add `cuenca.get_balance()` function helper

### CODE REVIEW

File: cuenca/__init__.py [hunk 1]
```diff
@@ -46,3 +47,8 @@
 from .version import __version__
 
 configure = session.configure
+
+
+def get_balance(session: Session = session) -> int:
```
Comment (matin) [inline cuenca/__init__.py:52]: This needs to be added to `__all__`

File: README.md [hunk 2]
```diff
@@ -98,6 +98,29 @@ transfers = cuenca.Transfer.all(
 # the total number of succeeded transfers
 count = cuenca.Transfer.count(status=Status.succeeded)
 ```
+## Balance
+
+### Current balance
```
Comment (matin) [inline README.md:103]: This is no longer necessary to show if the helper function exists

**Review by matin — CHANGES_REQUESTED** [review]
How is it possible that the two tests produce different results? That shouldn't be possible.

If you manually created the cassettes, you shouldn't do that. You should use mocks instead.

**Comment by felipaoo** [comment]
> How is it possible that the two tests produce different results? That shouldn't be possible.

I use two different api keys and users. The first one with transactions and `balance_entries` and the second one without `balance_entries`.

**Review by matin — APPROVED** [review]
looks good. ready for merge and release

**Pull request merged**